modules/react/icon/lib/AccentIcon.tsx

```diff
@@ -31,6 +34,7 @@ export interface AccentIconProps extends AccentIconStyles, Omit<IconProps, 'src'
   size?: number;
 }
 
+/** @deprecated */
```

Comment:
If this is deprecated, should we add more comments around why and when this happened? Also, should we add when this will be removed with the next version?